src/components/TalentsListPage/SearchPanel/SearchPanel.jsx

```diff
@@ -1,18 +1,11 @@
-import { useMemo } from "react";
 import styles from "./SearchPanel.module.scss";
-//import st from "../../Layout/Header/Header.module.scss";
+
+const inputNames = ["Job title", "Location", "Distance"];
+
 export function SearchPanel() {
-    const inputNames = useMemo(
-        () => [
-            { name: "Job title" },
-            { name: "Location" },
-            { name: "Distance" },
-        ],
-        []
-    );
     return (
         <div className={styles["search-panel"]}>
-            {inputNames.map(({ name }) => (
+            {inputNames.map((name) => (
                 <input
                     type="text"
                     className={styles["search-input"]}
```

Comment:
according to design these inputs have different width, so how would you apply different widths for these inputs if there is one specific class for every input? I said that you were better to separate these inputs into reusable component and use appropriate props to give through needed class or another selector. To avoid so difficultapproach in your task (you mustnt do this at all) just set this inputs without any cycle, just 3 tags "input", that is called "hardcoding"

Reply:
similar situation will be with handlers for these inputs, you will need to provide appropriate handler for each input